guest-tools/image/create-td-image.sh

```diff
@@ -43,13 +43,15 @@ fi
 
 LOGFILE=/tmp/tdx-guest-setup.txt
 FORCE_RECREATE=false
-OFFICIAL_UBUNTU_IMAGE=${OFFICIAL_UBUNTU_IMAGE:-"https://cloud-images.ubuntu.com/releases/noble/release/"}
-CLOUD_IMG=${CLOUD_IMG:-"ubuntu-24.04-server-cloudimg-amd64.img"}
+CODE_NAME=$(lsb_release -cs)
+UBUNTU_VERSION=$(lsb_release -rs)
+OFFICIAL_UBUNTU_IMAGE=${OFFICIAL_UBUNTU_IMAGE:-"https://cloud-images.ubuntu.com/releases/${CODE_NAME}/release/"}
```

Comment:
So for now it's only possible to create a guest image that matches the host OS, correct? I thought you mentioned wanting to support other guest OS's than may differ from the host OS? Is that something that will be enabled later?